TypeException: The function "foo" is not asynchronous (line 6, file "fixtures/compile/await/await.error.sync.ks")